velox/docs/functions/spark/string.rst

```diff
@@ -427,3 +427,36 @@ String Functions
     Returns string with all characters changed to uppercase. ::
 
         SELECT upper('SparkSql'); -- SPARKSQL
+
+.. spark:function:: varchar_type_write_side_check(string, limit) -> varchar
+
+    An implementation for Spark's varcharTypeWriteSideCheck function of class CharVarcharCodegenUtils.
```

Comment:
Perhaps remove the descriptions about Spark's detailed implementation and focus on the semantics of this function.